driver-core/src/main/com/mongodb/operation/ChangeStreamOperation.java

```diff
@@ -122,11 +123,22 @@ public FullDocument getFullDocument() {
     }
 
     /**
-     * Returns the logical starting point for the new change stream.
+     * Cache the logical starting point for the new change stream.
+     *
+     * @param resumeToken the resumeToken
+     * @return this
+     */
+    public ChangeStreamOperation<T> resumeToken(final BsonDocument resumeToken) {
```

Comment:
Do we need this new method?

Reply:
This method is used to cache the resume token from the ChangeStreamBatchCursor.

Reply:
`resumeToken` is the cached token that is updated when a new batch is received or when the ChangeStream is started. The spec shows the conditions when `resumeToken` should be updated (https://github.com/mongodb/specifications/blob/master/source/change-streams/change-streams.rst#id24). `resumeAfter` is the logical starting point for the change stream.

Reply:
Thanks